CHANGES.md

```diff
@@ -23,6 +23,18 @@
   `snare` informs users whose config contains `email` how to update to
   `errorcmd` to obtain the previous behaviour.
 
+## Minor changes
+
+* After daemonisation, all errors are now sent to syslog (previously a few
+  errors could still be sent to stderr).
+
+* Fix bug in parsing string escapes, where one character too many was
+  consumed after `\"`.
```

Comment:
Just to check, this is the intended rendering for that verbatim part:
![image](https://user-images.githubusercontent.com/604955/81802449-9ecf9200-9505-11ea-8082-acbc5d2d9750.png)

Correct?

Reply:
Yep!